impl/maven-cli/src/main/java/org/apache/maven/cling/utils/CLIReportingUtils.java

```diff
@@ -35,17 +35,6 @@
  *
  */
 public final class CLIReportingUtils {
-    // CHECKSTYLE_OFF: MagicNumber
-    public static final long MB = 1024 * 1024;
-
-    private static final long ONE_SECOND = 1000L;
-
-    private static final long ONE_MINUTE = 60 * ONE_SECOND;
-
-    private static final long ONE_HOUR = 60 * ONE_MINUTE;
```

Comment:
- https://github.com/openrewrite/rewrite-static-analysis/issues/554
assuming be circle issue as last block item was found. `ONE_DAY`